fe/fe-core/src/main/java/com/starrocks/planner/SetOperationNode.java

```diff
@@ -480,4 +482,45 @@ public boolean isVectorized() {
     public boolean canDoReplicatedJoin() {
         return false;
     }
+
+    @Override
+    public boolean pushDownRuntimeFilters(RuntimeFilterDescription description, Expr probeExpr) {
+        if (!probeExpr.isBoundByTupleIds(getTupleIds())) return false;
+
+        if (probeExpr instanceof SlotRef) {
+            boolean pushDown = false;
+            int probeSlotId = ((SlotRef) probeExpr).getSlotId().asInt();
+            Set<Integer> mappedProbeSlotIds = new HashSet<>();
+            for (Map<Integer, Integer> map : passThroughSlotMaps) {
+                if (map.containsKey(probeSlotId)) {
+                    mappedProbeSlotIds.add(map.get(probeSlotId));
+                }
+            }
+
+            for (int i = 0; i < materializedResultExprLists_.size(); i++) {
+                // try to push all children if any expr of a child can match `probeExpr`
+                for (Expr mexpr : materializedResultExprLists_.get(i)) {
+                    if ((mexpr instanceof SlotRef) && mappedProbeSlotIds.contains(((SlotRef) mexpr).getSlotId().asInt())) {
+                        mexpr.setUseVectorized(mexpr.isVectorized());
```

Comment:
```suggestion
                        mexpr.setUseVectorized(mexpr.isVectorized());
```

Reply:
What's the difference?

Reply:
Don't need the code~

Reply:
Ok.

Reply:
Could we just leave it here for now? Because it's a pattern used in `pushDownRuntimeFilters` in every Node class.  

Maybe we could remove it later in an batch way.